web/src/styles/components/buttons.scss

```diff
@@ -128,7 +128,12 @@
 }
 
 .btn-white {
+  &:hover {
+    @include addDropShadow($drop-shadow-md);
+  }
```

Comment:
[dust] interesting that you chose to include this code here rather than
something like `white: "usa-button text-info-darkest btn-white drop-shadow-md-on-hover",` (I think that this would also work right?)

Did you have a specific reason for favoring this approach?

Reply:
I thought that by incorporating it into the btn-white or landing-page-tile classes, we wouldn't have to be concerned about including the drop-shadow class when using them in other sections of the application.